pkg/cmd/recoveryapiserver/create.go

```diff
@@ -0,0 +1,122 @@
+package recoveryapiserver
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/client-go/tools/watch"
+
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/recovery"
+)
+
+type CreateOptions struct {
+	Options
+
+	KubeApiserverImage string
+	Timeout            time.Duration
```

Comment:
timeout of what?

Reply:
the whole create command, at this point the only asynchronous op is waiting for the server to come up